Packages/com.unity.inputsystem/InputSystem/Plugins/XR/Haptics/GetHapticCapabilitiesCommand.cs

```diff
@@ -1,31 +1,104 @@
 // ENABLE_VR is not defined on Game Core but the assembly is available with limited features when the XR module is enabled.
 #if UNITY_INPUT_SYSTEM_ENABLE_XR && (ENABLE_VR || UNITY_GAMECORE) || PACKAGE_DOCS_GENERATION
+using System;
 using System.Runtime.InteropServices;
 using UnityEngine.InputSystem.LowLevel;
 using UnityEngine.InputSystem.Utilities;
 
 namespace UnityEngine.InputSystem.XR.Haptics
 {
-    public struct HapticCapabilities
+    /// <summary>
+    /// Describes the haptic capabilities of a specific device.
+    /// </summary>
+    public readonly struct HapticCapabilities
     {
-        public HapticCapabilities(uint numChannels, uint frequencyHz, uint maxBufferSize)
+        /// <summary>
+        /// Initializes and returns an instance of <see cref="HapticCapabilities"/>.
+        /// </summary>
+        /// <param name="numChannels">The number of haptic channels available on this device.</param>
+        /// <param name="supportsImpulse">This device supports sending a haptic impulse.</param>
+        /// <param name="supportsBuffer">This device supports sending a haptic buffer.</param>
+        /// <param name="bufferFrequencyHz">The buffer frequency the device operates at in Hertz.</param>
+        /// <param name="bufferMaxSize">The max amount of buffer data that can be stored by the device.</param>
+        /// <param name="bufferOptimalSize">The optimal size of a device's buffer, taking into account frequency and latency.</param>
+        public HapticCapabilities(uint numChannels, bool supportsImpulse, bool supportsBuffer, uint bufferFrequencyHz, uint bufferMaxSize, uint bufferOptimalSize)
```

Comment:
Is this not a breaking change to remove the old constructor and add a new one with different parameters?